pkg/jwtmanager/jwtmanager.go

```diff
@@ -73,6 +74,72 @@ func populateSites() {
 	}
 }
 
+func decryptionKey() (interface{}, error) {
+	if strings.HasPrefix(cfg.Cfg.JWT.SigningMethod, "HS") {
+		return []byte(cfg.Cfg.JWT.Secret), nil
+	}
+
+	f, err := os.Open(cfg.Cfg.JWT.PublicKeyFile)
+	if err != nil {
+		return nil, fmt.Errorf("error opening Key %s: %s\n", cfg.Cfg.JWT.PrivateKeyFile, err)
```

Comment:
the error is for the PublicKeyFile but the error message is for the PrivateKeyFil